sass/layout/_front-layout.scss

```diff
@@ -33,15 +20,15 @@
 	font-size: 64px;
 	font-weight: 900;
 	margin: 0;
-	margin-top: -25px;
+	margin-top: -45px;
 	color: #263238;
 	text-align: center;
-	line-height: 60px;
 }
 
 .section-description {
+	font-family: "Raleway", sans-serif;
 	text-align: center;
-	padding: 0 30px;
+	padding: 5px 30px;
```

Comment:
I recommend coming up with a `$unit` var in sass that you can use as the base for your measurements so throughout your design things are consistent. ex: 

```
$unit: 5px; 

 .section-description {
  padding: $unit ($unit * 6); 
 ... 
```